acbuild/acbuild.go

```diff
@@ -239,3 +262,39 @@ func stdout(format string, a ...interface{}) {
 	out := fmt.Sprintf(format, a...)
 	fmt.Fprintln(os.Stdout, strings.TrimSuffix(out, "\n"))
 }
+
+func addACBuildAnnotation(cmd *cobra.Command, args []string) error {
+	const annoNamePattern = "appc.io/acbuild/command-%d"
```

Comment:
When I asked for this to be a const, my thinking was that it would be exported and essentially part of the acbuild API contract. But that might be premature, this is fine for now.